maquetado/setup/index.html

```diff
@@ -10,5 +10,12 @@
 </head>
 <body>
   <h1 class="title">Front-End Tools</h1>
+  <div class="tech">
+    <img src="assets/react_icon.svg" alt="ReactJS">
+    <h2>ReactJS</h2>
+    <p id="release">Release:2013</p>
```

Comment:
In wolox we try to avoid using `id` since it increases specificity a lot and it's harder to override. You can use a class instead. `tech-date`